cmake/external_mlir.cmake

```diff
@@ -20,8 +20,8 @@ set(MLIR_LLVM_REPO_URL https://github.com/llvm/llvm-project.git)
 set(MLIR_REPO_URL https://github.com/tensorflow/mlir.git)
 
 # Change these commit IDs to move to latest stable versions
-set(MLIR_LLVM_COMMIT_ID c0cad98)
-set(MLIR_COMMIT_ID 82d5084)
+set(MLIR_LLVM_COMMIT_ID db99d3a)
```

Comment:
Just curious, how do you get a matching LLVM_COMMIT_ID when you update MLIR? MLIR build instruction assume both are master HEAD.

Reply:
I do that. I move MLIR and LLVM commit ids to the current master and check that MLIR works.